### PR TITLE
updated receive use case description

### PR DESCRIPTION
**Pull request by andriimelnychuk**

### CODE REVIEW

File: source/includes/_usecases.md [hunk 1]
```diff
@@ -94,8 +94,13 @@ The [Affiliates Integration Guide](#affiliates-integration-guide) helps you get
 
 ## Receive Money
 
-You can receive money to the local bank details that come with your TransferWise account (AUD, EUR, GBP, NZD and USD) and reconcile these incoming payments via the API.
+You can receive money to the local bank details that come with your TransferWise account (USD, EUR, GBP, USD, NZD and PLN) and reconcile these incoming payments via the API.
 
+You’re also able to create a webhook subscription to receive information about incoming payments to your server. Here’s more information about the [webhooks](#webhook-events).
```
Comment (JamesFerretti) [inline source/includes/_usecases.md:99]: Slightly off wording, the information is going to your server, not the payments! Suggest:
> You’re also able to create a webhook subscription to receive event callbacks to your server when payments are received.

File: source/includes/_usecases.md [hunk 2]
```diff
@@ -94,8 +94,13 @@ The [Affiliates Integration Guide](#affiliates-integration-guide) helps you get
...
+
+Please note that we’re don't send any information over the webhooks that might contain Personally identifiable information (PII) about the sender (including the payment reference).
```
Comment (JamesFerretti) [inline source/includes/_usecases.md:101]: >Please note that we don't send any information over webhook calls that might contain personally identifiable information (PII) about the sender (including the payment reference).

File: source/includes/_usecases.md [hunk 3]
```diff
@@ -94,8 +94,13 @@ The [Affiliates Integration Guide](#affiliates-integration-guide) helps you get
 
 ## Receive Money
 
-You can receive money to the local bank details that come with your TransferWise account (AUD, EUR, GBP, NZD and USD) and reconcile these incoming payments via the API.
+You can receive money to the local bank details that come with your TransferWise account (USD, EUR, GBP, USD, NZD and PLN) and reconcile these incoming payments via the API.
```
Comment (amin-abutaleb-tw) [inline source/includes/_usecases.md:97]: USD is listed twice in that list

Comment (amin-abutaleb-tw) [reply source/includes/_usecases.md:97]: And missing AUD

**Pull request merged**